app/src/main/java/de/westnordost/streetcomplete/data/complete/CompleteTypes.java

```diff
@@ -0,0 +1,8 @@
+package de.westnordost.streetcomplete.data.complete;
+
+public class CompleteTypes
+{
+	public static final String
+			CHART = "chart",
```

Comment:
As I said above, chart may not be the best name. Maybe also "two-value answer" or so, as you can display it differently on the server side, of course.